### PR TITLE
[R5U-1283] Remove alias_method_chain for rails 5 compatibility

### PR DESCRIPTION
**Pull request by zenspider**
https://zendesk.atlassian.net/browse/R5U-1283

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,18 +3,7 @@ sudo: false
...
-  - 2.0.0
-  - 2.1.8
-  - 2.2.4
-  - 2.3.0
+  - 2.5.8
 gemfile:
-  - gemfiles/rails3.2.gemfile
-  - gemfiles/rails4.1.gemfile
   - gemfiles/rails4.2.gemfile
   - gemfiles/rails5.0.gemfile
```
Comment (mariozaizar) [inline .travis.yml:9]: Should we include 5.1, 5.2 ?

File: .travis.yml [hunk 2]
```diff
@@ -3,18 +3,7 @@ sudo: false
 bundler_args: ""
 cache: bundler
 rvm:
-  - 2.0.0
-  - 2.1.8
-  - 2.2.4
-  - 2.3.0
+  - 2.5.8
```
Comment (mariozaizar) [inline .travis.yml:6]: What about [2.6](https://github.com/zendesk/zendesk_channels/blob/master/.travis.yml#L6)?

File: .travis.yml (hunk 1 above)
Comment (zenspider) [reply .travis.yml:9]: No, not yet. There was some hardcoded top level versions. Prolly nothing, but I'd rather address that out of band when we start on 5.1

File: .travis.yml (hunk 2 above)
Comment (zenspider) [reply .travis.yml:6]: We're not testing classic against 2.6 yet, so I only added this one.

Based on the code here, 2.6 and 2.7 will be fine. This is really simple stuff.

File: .travis.yml (hunk 1 above)
Comment (grosser) [reply .travis.yml:9]: 5.2 would be nice ... 5.0 can go away once we no longer use it :)

**Comment by grosser** [comment]
gemspec should say   `s.required_ruby_version = '>= 2.5'`

**Comment by zenspider** [comment]
Good catch. I need to bump the minimum version for actionpack in the gemspec as well.

**Comment by zenspider** [comment]
Extra version goodness added...

**Comment by zenspider** [comment]
@grosser can you do the honors or should I poke Pierre?

**Pull request merged**